mappings/net/minecraft/block/material/Material.mapping

```diff
@@ -0,0 +1,61 @@
+CLASS none/ayo net/minecraft/block/material/Material
+	CLASS none/ayo$1
+		METHOD c isSolid ()Z
+	FIELD A CACTUS Lnone/ayo;
+	FIELD B OTHER Lnone/ayo;
+	FIELD C PLANT_BLOCK Lnone/ayo;
```

Comment:
why not map the plant mapping as crop and change plant block to plant?

On Thu, Aug 18, 2016 at 10:16 AM, copygirl <notifications@github.com> wrote:

> In mappings/net/minecraft/block/material/Material.mapping
> <https://github.com/FabricMC/pomf/pull/19#discussion_r75349442>:
>
> > @@ -0,0 +1,61 @@
> > +CLASS none/ayo net/minecraft/block/material/Material
> > +	CLASS none/ayo$1
> > +		METHOD c isSolid ()Z
> > +	FIELD A CACTUS Lnone/ayo;
> > +	FIELD B OTHER Lnone/ayo;
> > +	FIELD C PLANT_BLOCK Lnone/ayo;
>
> Essentially, PLANT_BLOCK is meant for full blocks: Melon and pumpkin. I
> addressed this in the notes, and I agree.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/FabricMC/pomf/pull/19/files/a1c50ce07a1e7f2bac06e3a3d2b0e06b72d6e7c7#r75349442>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABusJJKdIkOgb_QvCH7sdyMO8_rrnQh6ks5qhJNugaJpZM4JnthW>
> .
>